plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/action/CompareWithBranchAction.java

```diff
@@ -33,7 +33,7 @@ public CompareWithBranchAction(BranchListPresenter presenter,
                                    AppContext appContext,
                                    GitLocalizationConstant locale,
                                    ProjectExplorerPresenter projectExplorer) {
-        super(locale.compareWithBranchTitle(), locale.compareWithBranchTitle(), null, appContext, projectExplorer);
+        super(locale.compareWithBranchTitle(), locale.compareWithBranchTitle(), (String)null, appContext, projectExplorer);
```

Comment:
An extra constructor added to base org.eclipse.che.ide.ext.git.client.action.GitAction